### PR TITLE
Init boba chain

### PR DESCRIPTION
**Pull request by Hosuke**
Init boba chain

### CODE REVIEW

**Comment by Hosuke** [comment]
prices automation CI ❌ can be ignored in this PR.

File: sources/_base_sources/evm/boba_base_sources.yml
Comment (jeff-dude) [inline sources/_base_sources/evm/boba_base_sources.yml:1]: i assume this source list is shorter due to decoded not available yet, just raw data?
example, returns no rows:
```
select * from boba.logs_decoded limit 10
```

Comment (Hosuke) [reply sources/_base_sources/evm/boba_base_sources.yml:1]: Yes, I can put them as placeholders first, as they existed in the namespace.

Comment (Hosuke) [inline sources/_base_sources/evm/boba_base_sources.yml:1]: I have tried to match all raw tables and schema with https://dune.com/data?category=canonical&namespace=boba

Comment (Hosuke) [reply sources/_base_sources/evm/boba_base_sources.yml:1]: Updated now, this PR should be ready to go.

**Review by jeff-dude — APPROVED** [review]
thank you. we can iterate and add more tokens as needed when adding projects to spellbook and can dig into the data a bit further

**Pull request merged**